tfhe/src/high_level_api/compact_list.rs

```diff
@@ -366,27 +370,27 @@ pub struct CompactCiphertextListExpander {
     tag: Tag,
 }
 
-impl CompactCiphertextListExpander {
-    pub fn len(&self) -> usize {
+impl CiphertextList for CompactCiphertextListExpander {
```

Comment:
I'm not a big fan of having to load prelude to be able to use structs, but on the other hand it would lead to code duplication (having the code on the struct and a trait forwarding to those), I guess we just have to tell people to always import the prelude for the HL stuff, which is what we do I believe in the docs